inferno-ml-server-types/inferno-ml-server-types.cabal

```diff
@@ -36,33 +36,38 @@ library
     Inferno.ML.Server.Client
     Inferno.ML.Server.Client.Bridge
     Inferno.ML.Server.Types
+    Inferno.ML.Types.Value
 
   hs-source-dirs:  src
   build-depends:
     , aeson
     , attoparsec
     , base
+    , bson
     , bytestring
     , conduit
     , containers
     , deepseq
     , extra
     , generic-lens
     , hashable
+    , hasktorch
```

Comment:
Unfortunately we can't use `hasktorch` as a dependency without dragging the whole thing into `all`. We're on incompatible GHC versions, and `inferno-ml-server-types` is a dependency in `all`